src/deepsparse/server/cli.py

```diff
@@ -166,30 +106,185 @@ def config(
         "Defaults to half of the num_cores set"
     ),
 )
-@click.option(
-    "--host",
-    type=str,
-    default="0.0.0.0",
+
+INTEGRATION_OPTION = click.option(
+    "--integration",
+    type=click.Choice(["local", "sagemaker"], case_sensitive=False),
+    default="local",
     help=(
-        "Bind socket to this host. Use --host 0.0.0.0 to make the application "
-        "available on your local network. "
-        "IPv6 addresses are supported, for example: --host '::'."
+        "Name of deployment integration that this server will be deployed to "
+        "Currently supported options are 'default' and 'sagemaker' for "
+        "inference deployment with Amazon Sagemaker"
+    ),
+)
+
+
+@click.group(
+    invoke_without_command=True,
+    context_settings=dict(
+        token_normalize_func=lambda x: x.replace("-", "_"), show_default=True
     ),
 )
 @click.option(
-    "--port",
-    type=int,
-    default=5543,
-    help="Bind to a socket with this port.",
+    "--config_file",
+    type=str,
+    default=None,
+    help="Configuration file containing info on how to serve the desired models.",
 )
 @click.option(
-    "--log-level",
-    type=click.Choice(
-        ["debug", "info", "warn", "critical", "fatal"], case_sensitive=False
+    "--task",
+    type=click.Choice(SupportedTasks.task_names(), case_sensitive=False),
+    default=None,
+    help="The task the model_path is serving.",
+)
+@HOST_OPTION
+@PORT_OPTION
+@LOG_LEVEL_OPTION
+@HOT_RELOAD_OPTION
+@MODEL_OPTION
+@BATCH_OPTION
+@CORES_OPTION
+@WORKERS_OPTION
+@INTEGRATION_OPTION
+@click.pass_context
+def main(
+    ctx,
+    task: Optional[str],
+    config_file: Optional[str],
+    host: str,
+    port: int,
+    log_level: str,
+    hot_reload_config: bool,
+    model_path: str,
+    batch_size: int,
+    num_cores: int,
+    num_workers: int,
+    integration: str,
+):
+    """
+    Start a DeepSparse inference server for serving the models and pipelines.
```

Comment:
let's update help text to include the pathway w/o the named args

Reply:
Updated